autogluon/utils/tabular/ml/models/tab_transformer/TabTransormerEncoder.py

```diff
@@ -0,0 +1,660 @@
+
```

Comment:
This entire file seems to be very disconnected to the actual TabTransformer model.

Can we avoid adding unused classes/functions here, especially since it seems they depend on torch.